test/Atmos/Parameterizations/Microphysics/runtests.jl

```diff
@@ -1,161 +1,12 @@
-            param_set,
-            PhasePartition(0.0, q_liq_sat, 0.0),
-            PhasePartition(0.0, q_liq, 0.0),
-        ) ≈ (1 - fr) * q_liq_sat / τ_cond_evap(param_set)
-    end
-end
-
-@testset "RainAutoconversion" begin
-
-    _q_liq_threshold = q_liq_threshold(param_set)
-    _τ_acnv = τ_acnv(param_set)
-    q_liq_small = 0.5 * _q_liq_threshold
-    @test conv_q_liq_to_q_rai_acnv(param_set, q_liq_small) == 0.0
-
-    q_liq_big = 1.5 * _q_liq_threshold
-    @test conv_q_liq_to_q_rai_acnv(param_set, q_liq_big) ==
-          0.5 * _q_liq_threshold / _τ_acnv
-end
-
-@testset "RainAccretion" begin
-
-    # eq. 5b in Smolarkiewicz and Grabowski 1996
-    # https://doi.org/10.1175/1520-0493(1996)124<0487:TTLSLM>2.0.CO;2
-    function accretion_empir(q_rai::FT, q_liq::FT, q_tot::FT) where {FT <: Real}
-        rr = q_rai / (FT(1) - q_tot)
-        rl = q_liq / (FT(1) - q_tot)
-        return FT(2.2) * rl * rr^FT(7 / 8)
-    end
-
-    # some example values
-    q_rain_range = range(1e-8, stop = 5e-3, length = 10)
-    ρ_air, q_liq, q_tot = 1.2, 5e-4, 20e-3
-
-    for q_rai in q_rain_range
-        @test conv_q_liq_to_q_rai_accr(param_set, q_liq, q_rai, ρ_air) ≈
-              accretion_empir(q_rai, q_liq, q_tot) atol =
-            0.1 * accretion_empir(q_rai, q_liq, q_tot)
-    end
-end
-
-@testset "RainEvaporation" begin
-
-    # eq. 5c in Smolarkiewicz and Grabowski 1996
-    # https://doi.org/10.1175/1520-0493(1996)124<0487:TTLSLM>2.0.CO;2
-    function rain_evap_empir(
-        param_set::AbstractParameterSet,
-        q_rai::FT,
-        q::PhasePartition,
-        T::FT,
-        p::FT,
-        ρ::FT,
-    ) where {FT <: Real}
-
-        ts_neq = TemperatureSHumNonEquil(param_set, T, ρ, q)
-        q_sat = q_vap_saturation(ts_neq)
-        q_vap = q.tot - q.liq
-        rr = q_rai / (1 - q.tot)
-        rv_sat = q_sat / (1 - q.tot)
-        S = q_vap / q_sat - 1
-
-        ag, bg = FT(5.4 * 1e2), FT(2.55 * 1e5)
-        G = FT(1) / (ag + bg / p / rv_sat) / ρ
-
-        av, bv = FT(1.6), FT(124.9)
-        F =
-            av * (ρ / FT(1e3))^FT(0.525) * rr^FT(0.525) +
-            bv * (ρ / FT(1e3))^FT(0.7296) * rr^FT(0.7296)
-
-        return 1 / (1 - q.tot) * S * F * G
-    end
-
-    # example values
-    T, p = 273.15 + 15, 90000.0
-    ϵ = 1.0 / molmass_ratio(param_set)
-    p_sat = saturation_vapor_pressure(param_set, T, Liquid())
-    q_sat = ϵ * p_sat / (p + p_sat * (ϵ - 1.0))
-    q_rain_range = range(1e-8, stop = 5e-3, length = 10)
-    q_tot = 15e-3
-    q_vap = 0.15 * q_sat
-    q_ice = 0.0
-    q_liq = q_tot - q_vap - q_ice
-    q = PhasePartition(q_tot, q_liq, q_ice)
-    R = gas_constant_air(param_set, q)
-    ρ = p / R / T
-
-    for q_rai in q_rain_range
-        @test conv_q_rai_to_q_vap(param_set, q_rai, q, T, p, ρ) ≈
-              rain_evap_empir(param_set, q_rai, q, T, p, ρ) atol =
-            -0.5 * rain_evap_empir(param_set, q_rai, q, T, p, ρ)
-    end
+    runmpi(tests, @__FILE__)
```

Comment:
@charleskawczynski - could you take a look at this line? 

I'm getting an error from windows on azure:
```
Microphysics tests: Error During Test at D:\a\1\s\test\Atmos\Parameterizations\Microphysics\runtests.jl:4
Got exception outside of a @test
MethodError: no method matching runmpi(::Array{Tuple{Int64,String},1}, ::String)
Closest candidates are:
runmpi(::Any; ntasks) at D:\a\1\s\test\testhelpers.jl:4
```
I just wanted to have one `runtest.jl` file here to start my unit and integration tests.

Reply:
Looks like the path to `include(joinpath("../../../", "testhelpers.jl"))` is not correct (`runmpi` is defined in `testhelpers.jl`).

Reply:
Thats strange `../../../testhelpers.jl` is exactly where the file is located related to where the `runtests.jl` file is.

Also, this error only happens on windows?

Reply:
I think you may need `include(joinpath(@__DIR__, "..","..","..", "testhelpers.jl"))` if it's relative to this folder's path, otherwise including this file depends on where you launch Julia from.

Reply:
Also, `/` is a OS-dependent path separator, so we should always use `joinpath`.